server/lib/userlib.js

```diff
@@ -45,24 +13,4 @@ export default {
     }
     return null;
   },
-
-  updateUserInfoFromClearbit(user) {
-    // don't try to fetch user details if user hasn't provided a first name (i.e. if they wanted to remain incognito)
-    if (!user.email || !user.firstName) {
-      return Promise.resolve();
-    }
-    return this.getUserData(user.email).then(userData => {
-      if (userData) {
-        user.firstName = user.firstName || userData.name.givenName;
-        user.lastName = user.lastName || userData.name.familyName;
```

Comment:
We're only setting `lastName` if `firstName` was previously sent and `lastName` was empty. Doesn't make sense!